common/templates/form-wizard.njk

```diff
@@ -66,10 +66,12 @@
     {% endblock %}
 
     {% block formActions %}
-      {{ govukButton({
-        text: t(options.buttonText or "actions::continue"),
-        classes: options.buttonClasses
-      }) }}
+      {% if not hideContinueButton %}
+        {{ govukButton({
+          text: t(options.buttonText or "actions::continue"),
+          classes: options.buttonClasses
+        }) }}
+      {% endif %}
```

Comment:
`hideContinueButton` is used when you get no search results and the only journey you can take is back. Felt a bit dirty but this seemed the easiest way to match the wireframe.